clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -313,7 +313,7 @@ public void onFailure(RuntimeException e) {
                 metadata());
 
         // create the request for the coordinator
-        log.debug("Issuing request ({}: {}) to coordinator {}", ApiKeys.JOIN_GROUP, request, this.coordinator.id());
+        log.debug("Sending JoinGroup ({}) to coordinator {}", request, this.coordinator.id());
```

Comment:
Usually the host:port is more helpful when debugging, and it's kind of a pain to do the reverse mapping from id to host in my head. The only time the id is helpful is when looking at NetworkClient issues.